src/e2e/mock-docker.sh

```
@@ -197,9 +224,17 @@ exec)
     *)
-      # General command: run in exec_cwd
+      # General command: rewrite container-absolute paths into rootfs paths
+      rewritten_args=()
+      for arg in "$@"; do
+        if [[ "$arg" == /* ]]; then
+          rewritten_args+=("$rootfs$arg")
+        else
+          rewritten_args+=("$arg")
+        fi
+      done
       cd "$exec_cwd"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)